alfie.py

```diff
@@ -1,53 +1,169 @@
+                         PANDOC_EXTENTIONS)
 
 app = Quart(__name__)
 app.config['SECRET_KEY'] = os.environ['ALFIE_SECRET']
 
-Markdown(app)
-
 projects = {'icepap-ipassign':
-            {'README': '/home/cydanil/icepap-ipassign/README.md',
-             'GUI README': '/home/cydanil/icepap-ipassign/ipa_gui/gui.md'},
+            {'README': ('/home/cydanil/icepap-ipassign/README.md',
+                        'Complete ipassign documentation'),
+             'GUI README': ('/home/cydanil/icepap-ipassign/ipa_gui/gui.md',
+                            'Qt Gui development documentation')},
             'alfie':
-            {'Github': 'https://github.com/cydanil/alfie'},
+            {'Page': ('/home/cydanil/Downloads/Alfie.html',
+                      'Alfie homepage')},
             'h5py':
-            {'Groups': '/home/cydanil/h5py/docs/high/group.rst',
-             'Files': '/home/cydanil/h5py/docs/high/file.rst',
-             'Build': '/home/cydanil/h5py/docs/build.rst'},
-            'rook': {},
-            'seagull': {},
+            {'Groups': ('/home/cydanil/h5py/docs/high/group.rst',
+                        'hdf5 groups manual'),
+             'Files': ('/home/cydanil/h5py/docs/high/file.rst',
+                       'hdf5 files manual'),
+             'Build': ('/home/cydanil/h5py/docs/build.rst',
+                       'hdf5 build how-to')},
+            'librashpa':
+            {'librashpa.pdf': ('/home/cydanil/alfie/site/librashpadoc/librashpadoc.pdf',
+                               'librashpa output pdf documentation'),
+             'librashpa.html': ('/home/cydanil/alfie/site/librashpadoc/html/index.html',
+                                'librashpa html documentation')},
+            'rook':
+            {'Sample docx file': ('/home/cydanil/alfie/tests/test_data/file-sample_1MB.docx',
+                                  'Test microsoft docx file handling'),
+             'Sample doc file': ('/home/cydanil/alfie/tests/test_data/file-sample_1MB.doc',
+                                 'Test legacy doc file handling')},
+            'seagull': {'WHIST CE': ('https://alfresco.esrf.fr/share/page/site/ce-certification-wg/document-details?nodeRef=workspace://SpacesStore/789b28d6-6111-4fbd-b7dc-545406886f26',
+                                     'WHIST certification documentation (alfresco)')},
             'bluejay': {},
             'pelican': {},
             'ostrich': {},
             }
 
 
 @app.route('/')
+@app.route('/index')
 async def index():
     html = await render_template('index.html', projects=projects)
     return html
 
 
+@app.route('/create', methods=['POST'])
+async def create():
+    """Create a new project in the database."""
+
+    form = await request.form
+    name = form.get('name')
+
+    if name is None:
+        await flash('Did not specify a project name')
+    elif name in projects:
+        await flash('This project already exists')
+    else:
+        projects[name] = {}
```

Comment:
Because entries are not sorted in any particular fashion (yet), emit a flash saying that the project was successfully added.